templates/frontOffice/default/layout.tpl

```diff
@@ -268,7 +268,11 @@ GNU General Public License : http://www.gnu.org/licenses/
     {block name="after-javascript-include"}{/block}
 
     {hook name="main.javascript-initialization"}
-
+    <script>
+       // fix path for addCartMessage
+       // if you use '/' in your URL rewriting, the cart message is not displayed

```

Comment:
A useful comment would be : "Define the addCartMessage url, used in thelia.js to update th mini-cart content.", or something meaningful.